Makefile

```diff
@@ -100,7 +100,7 @@ local-status: ## Show the status of the containers in the dev environment.
 
 .PHONY: local-rebuild
 local-rebuild: .env.ecr local-ecr-login ## Rebuild local dev without re-importing data
-	docker-compose $(COMPOSE_OPTS) build frontend backend processing wmg_processing
+	docker-compose $(COMPOSE_OPTS) build frontend backend processing wmg_processing database oidc localstack
```

Comment:
`database oidc localstack` should start up without explicitly mentioning them since collectively`frontend backend processing and wmg_processing` depend on those containers.